Lecture11.md

```diff
@@ -0,0 +1,63 @@
+# February 22, 2023 
+## _New Testing Metrics_
+### Objective: Today we learn about different kinds of testing metrics that we can use to ensure the best quality.
+***
+Review from last class:
++ **Unit Testing** is when you test  indivual chunks of source code to determine that it works in isolation from the rest of the program. Some adavntages of this testing metric is as follows:
+* since it is in small chunks it's easier to understand what is going on in that specfic section.
+* we test features in isolation so it can be easier to locate a bug.
++ because it is small testing becomes faster.
+
+*** 
++ building on this idea, we can begin to discuss something called: **Test Driven Development**
+**Test Driven Development** is when you write the test file before you even write the new feature. Which works great in conversation with unit testing. For example:
++ Write a unit test for a new feature 
+    + _your test will fail at this point
```

Comment:
The "_" appear to be a type and appears in the md file as -_your test will fail at this point